utils_nlp/models/transformers/sequence_classification.py

```diff
@@ -1,2 +1,117 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License.
\ No newline at end of file
+# Licensed under the MIT License.
+
+import torch
+from pytorch_transformers.modeling_bert import (
+    BERT_PRETRAINED_MODEL_ARCHIVE_MAP,
+    BertForSequenceClassification,
+)
+from pytorch_transformers.modeling_distilbert import (
+    DISTILBERT_PRETRAINED_MODEL_ARCHIVE_MAP,
+    DistilBertForSequenceClassification,
+)
+from pytorch_transformers.modeling_roberta import (
+    ROBERTA_PRETRAINED_MODEL_ARCHIVE_MAP,
+    RobertaForSequenceClassification,
+)
+from pytorch_transformers.modeling_xlnet import (
+    XLNET_PRETRAINED_MODEL_ARCHIVE_MAP,
+    XLNetForSequenceClassification,
+)
+from torch.utils.data import DataLoader, DistributedSampler, RandomSampler, TensorDataset
+
+from utils_nlp.dataset.pytorch import SCDataSet
+from utils_nlp.models.transformers.common import MAX_SEQ_LEN, TOKENIZER_CLASS, fine_tune
+
+MODEL_CLASS = {}
+MODEL_CLASS.update({k: BertForSequenceClassification for k in BERT_PRETRAINED_MODEL_ARCHIVE_MAP})
+MODEL_CLASS.update(
+    {k: RobertaForSequenceClassification for k in ROBERTA_PRETRAINED_MODEL_ARCHIVE_MAP}
+)
+MODEL_CLASS.update({k: XLNetForSequenceClassification for k in XLNET_PRETRAINED_MODEL_ARCHIVE_MAP})
+MODEL_CLASS.update(
+    {k: DistilBertForSequenceClassification for k in DISTILBERT_PRETRAINED_MODEL_ARCHIVE_MAP}
+)
+
+
+def _list_supported_models():
+    return list(MODEL_CLASS)
+
+
+def create_dataset_from_df(df, text_col, label_col):
+    return SCDataSet(df, text_col, label_col)
+
+
+class Processor:
+    def __init__(self, pt_model_name, tokenize=None, to_lower=False, cache_dir="."):
```

Comment:
IMHO, "pt" adds additional comprehension overhead. "Point" is my first guess, then pretrained, then  pytorch-transformers. I think "model_name" would be good enough. It's up to you. I will follow whatever you decide on.